bindings/python/README.md

```diff
@@ -0,0 +1,83 @@
+# PyMongoArrow
+
+- Info: A companion library to PyMongo that makes it easy to move data between
```

Comment:
Is there a nicer way to render this info rather than a bulleted list? The rst version was in a table.

Reply:
GFM does not support tables with no headers.

Reply:
How about we just remove this section?

Reply:
Done, I've opened https://jira.mongodb.org/browse/ARROW-196 to track adding badges instead.